community_server/src/Controller/AppController.php

```diff
@@ -88,19 +88,8 @@ public function initialize()
         // load current balance
         $session = $this->getRequest()->getSession();
         $state_user_id = $session->read('StateUser.id');
-        if ($state_user_id) {
-            $stateBalancesTable = TableRegistry::getTableLocator()->get('stateBalances');
-            $stateBalanceQuery = $stateBalancesTable
-                  ->find('all')
-                  ->contain(false)
-                  ->where(['state_user_id' => $state_user_id]);
-            if ($stateBalanceQuery->count() == 1) {
-              //var_dump($stateBalanceEntry->first());
-                $session->write('StateUser.balance', $stateBalanceQuery->first()->decay);
-              //echo "stateUser.balance: " . $session->read('StateUser.balance');
-            }
-        }
-
```

Comment:
It was for loading the balance for old frontend on startup to show it in the menu. 
But in the meantime this was moved to requestLogin and this code here wasn't longer needed. 
It is also wrong, because it don't take into account that decay starting after the decay block.

Reply:
In routes was an error

Reply:
And the Community Server Dockerfile I have changed to get coverage working on community server